.circleci/config.yml

```diff
@@ -1,6 +1,15 @@
 version: 2
+
+workflows:
+  version: 2
+  test:
+    jobs:
+      - build-python-3.6
+      - build-python-3.7
+      - build-python-3.8
```

Comment:
Autobuilds ran on my branch: https://github.com/jacksonllee/wikipron/commits/repo-maintenance

Now we're running a separate job for each Python version we support:
<img width="508" alt="Screen Shot 2019-09-06 at 3 34 56 PM" src="https://user-images.githubusercontent.com/6993153/64458816-f5a1e380-d0bb-11e9-849d-397aed615046.png">